osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -79,64 +73,98 @@ public ScoresContainer()
+            if (api.LocalUser.Value.IsSupporter)
+                AddRange(new Drawable[]
+                {
+                    scopeSelector = new LeaderboardScopeSelector(),
+                    modSelector = new LeaderboardModSelector()
+                });
+
+            Add(new Container
+            {
+                AutoSizeAxes = Axes.Y,
+                RelativeSizeAxes = Axes.X,
+                Children = new Drawable[]
+                {
+                    new FillFlowContainer
                     {
-                        topScoresContainer = new FillFlowContainer
-                        {
-                            RelativeSizeAxes = Axes.X,
-                            AutoSizeAxes = Axes.Y,
-                            Direction = FillDirection.Vertical,
-                            Spacing = new Vector2(0, 5),
-                        },
-                        scoreTable = new ScoreTable
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Width = 0.95f,
+                        Direction = FillDirection.Vertical,
+                        Spacing = new Vector2(0, spacing),
+                        Children = new Drawable[]
                         {
-                            Anchor = Anchor.TopCentre,
-                            Origin = Anchor.TopCentre,
+                            topScoresContainer = new FillFlowContainer
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Vertical,
+                                Spacing = new Vector2(0, 5),
+                            },
+                            scoreTable = new ScoreTable
+                            {
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                            }
                         }
+                    },
+                    loadingAnimation = new LoadingAnimation
+                    {
+                        Alpha = 0,
                     }
-                },
-                loadingAnimation = new LoadingAnimation
-                {
-                    Alpha = 0,
-                    Margin = new MarginPadding(20),
-                },
-            };
-        }
+                }
+            });
 
-        [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
-        {
-            background.Colour = colours.Gray2;
+            scopeSelector?.Current.BindValueChanged(scope => getScores(scope.NewValue, modSelector.SelectedMods.Value));
+            modSelector?.SelectedMods.BindValueChanged(mods => getScores(scopeSelector.Current.Value, mods.NewValue));
+            Beatmap.BindValueChanged(beatmap =>
```

Comment:
why is `Beatmap` changed into a bindable from a property, it isn't bound to anything, can be reverted back into a property.

Reply:
Check `BeatmapSetOverlay`

Reply:
Doesn't seem to be necessary to bind it to `Header.Picker.Beatmap` directly, leave it the same as it was

Reply:
We want to use bind flow as much as possible. In the future other objects in beatmap set will use Bindable instead of property